gazebo_plugins/include/gazebo_plugins/gazebo_ros_bumper.h

```diff
@@ -66,16 +67,21 @@ namespace gazebo
     /// \brief pointer to ros node
     private: ros::NodeHandle* rosnode_;
     private: ros::Publisher contact_pub_;
+    private: ros::Publisher locus_bump_pub_;
```

Comment:
Maybe something like `bump_bool_pub_`? No reason why this has to be `locus_whatever`.